desktop/SmesherMetadataService.ts

```diff
@@ -0,0 +1,69 @@
+import path from 'path';
+import {
+  deleteFileAsync,
+  isFileExists,
+  readFileAsync,
+  writeFileAsync,
+} from './utils';
+
+interface SmesherMetadata {
+  smeshingStart?: number;
+  posInitStart?: number;
+}
+class SmesherMetadataService {
```

Comment:
Seems we don't need a Class here.
Much better will be to have just a set of functions.
Isn't it?